rackcity/views/asset_views.py

```diff
@@ -319,13 +327,14 @@ def asset_modify(request):
                 asset_id=existing_asset.id
             )
         except MacAddressException as error:
-            failure_message += "Some mac addresses couldn't be saved. " + \
+            failure_message = "Some mac addresses couldn't be saved. " + \
```

Comment:
this was `+=` before from your recent PR but failure_message isn't actually defined yet

Reply:
Ooh good catch thank you